src/sql/workbench/browser/modelComponents/checkbox.component.ts

```diff
@@ -93,6 +95,7 @@ export default class CheckBoxComponent extends ComponentBase implements ICompone
 		if (this.required) {
 			this._input.required = this.required;
 		}
+		this.validate();
```

Comment:
Note that this is leaving a dangling promise. But unfortunately due to validate returning a Thenable instead of a Promise we can't use catch directly and I didn't see the value in wrapping this in a promise to handle it correctly since ModelView in general isn't handling this fully in most cases anyways and there's nothing useful we can do if it fails regardless.